packages/apps/tools/src/components/Global/AddNetworkModal/AddNetworkModal.tsx

```diff
@@ -91,37 +91,31 @@ export const AddNetworkModal: FC<IAddNetworkModalProps> = (props) => {
                   <Stack direction="column" gap="$sm">
                     <TextField
                       label={t('Network label')}
-                      inputProps={{
-                        id: 'label',
-                        ...register('label'),
-                        onChange: (e) => setLabel(e.target.value),
-                        value: label,
-                        placeholder: 'devnet',
-                      }}
+                      id="label"
+                      {...register('label')}
```

Comment:
I know it was there before but I wonder why use `register` in this case since you also use the component in controlled by passing `value` and `onChange`, I would recommend removing them and use `register` only, same for the other fields

Reply:
@sanderlooijenga this change is actually how it would work without using the forwardRef, is this an okay change for tools?